sys/include/crypto/psa/riot_ciphers.h

```diff
@@ -38,8 +38,9 @@ typedef cipher_t psa_cipher_aes_256_ctx_t;
 #if IS_USED(MODULE_PSA_RIOT_CIPHER_CHACHA20)
 #include "crypto/chacha20poly1305.h"
 typedef struct {
-    uint8_t is_finished;
-    union context {
+    uint8_t buffer[64];
```

Comment:
isn't there some macro in `chacha20poly1305.h` that returns the block size?

Reply:
no, not at the moment